src/gradualizer_prelude.erl

```diff
@@ -2,8 +2,70 @@
 
 %% This module contains specs to replace incorrect or inexact specs in OTP.
 
+-type ext_binary() :: binary().
```

Comment:
Why not just write `binary()` where this one is used?

Reply:
The reason I kept the `ext_binary` name is that it is used in the documentation to indicate a special form of binary. But for Gradualizer it doesn't really matter. I'll change it.